src/topp/MSGFPlusAdapter.cpp

```diff
@@ -162,7 +164,7 @@ class MSGFPlusAdapter :
     registerStringOption_("instrument", "<choice>", instruments_[0], "Instrument that generated the data ('low_res'/'high_res' refer to LCQ and LTQ instruments; MS-GF+ parameter '-inst')", false);
     setValidStrings_("instrument", instruments_);
 
-    registerStringOption_("enzyme", "<choice>", enzymes_[1], "Enzyme used for digestion, or type of cleavage (MS-GF+ parameter '-e')", false);
+    registerStringOption_("enzyme", "<choice>", enzymes_[6], "Enzyme used for digestion, or type of cleavage (MS-GF+ parameter '-e')", false);
```

Comment:
nice PR!
Could you add a comment here like:
"Note: msgf+ only supports '/P' cutting rules."